daemon/storage/windows/windows.go

```diff
@@ -20,27 +20,27 @@ import (
 	"time"
 	"unsafe"
 
-	"github.com/Microsoft/go-winio"
+	winio "github.com/Microsoft/go-winio"
```

Comment:
hum, it shouldn't matter, the package name is `winio` anyway (https://github.com/Microsoft/go-winio/blob/master/file.go, …)